cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/Builtins.java

```diff
@@ -0,0 +1,132 @@
+            "cmake_policy",
+            "configure_file",
+            "create_test_sourcelist",
+            "define_property",
+            "elseif",
+            "else",
+            "enable_language",
+            "enable_testing",
+            "endforeach",
+            "endfunction",
+            "endif",
+            "endmacro",
+            "endwhile",
+            "execute_process",
+            "export",
+            "file",
+            "find_file",
+            "find_library",
+            "find_package",
+            "find_path",
+            "find_program",
+            "fltk_wrap_ui",
+            "foreach",
+            "function",
+            "get_cmake_property",
+            "get_directory_property",
+            "get_filename_component",
+            "get_property",
+            "get_source_file_property",
+            "get_target_property",
+            "get_test_property",
+            "if",
+            "include_directories",
+            "include_external_msproject",
+            "include_regular_expression",
+            "include",
+            "install",
+            "link_directories",
+            "list",
+            "load_cache",
+            "load_command",
+            "macro",
+            "mark_as_advanced",
+            "math",
+            "message",
+            "option",
+            "project",
+            "qt_wrap_cpp",
+            "qt_wrap_ui",
+            "remove_definitions",
+            "return",
+            "separate_arguments",
+            "set_directory_properties",
+            "set_property",
+            "set",
+            "set_source_files_properties",
+            "set_target_properties",
+            "set_tests_properties",
+            "site_name",
+            "source_group",
+            "string",
+            "target_compile_definitions",
+            "target_compile_options",
+            "target_include_directories",
+            "target_link_libraries",
+            "try_compile",
+            "try_run",
+            "unset",
+            "variable_watch",
+            "while",
+
+            // added by hand
+            "cmake_parse_arguments"
+    );
+
+    /** @see <a href='https://cmake.org/cmake/help/v3.0/manual/cmake-commands.7.html#deprecated-commands'>cmake help</a> */
+    List<String> DEPRECATED = Arrays.asList("build_name",
+            "exec_program",
+            "export_library_dependencies",
+            "install_files",
+            "install_programs",
+            "install_targets",
+            "link_libraries",
+            "make_directory",
+            "output_required_files",
+            "remove",
+            "subdir_depends",
+            "subdirs",
+            "use_mangled_mesa",
+            "utility_source",
+            "variable_requires",
+            "write_file");
+
+    /** @see <a href='https://cmake.org/cmake/help/v3.0/manual/cmake-commands.7.html#ctest-commands'>cmake help</a> */
+    List<String> CTEST = Arrays.asList("ctest_build",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "CTEST" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00116)